pom.xml

```diff
@@ -94,6 +94,10 @@
         <module>search/connector</module>
         <module>distro</module>
         <module>docs</module>
+        <module>storage/infinispan</module>
+        <module>storage/jpa</module>
+        <module>storage/kafka</module>
+        <module>storage/streams</module>
```

Comment:
I don't think we can do this until `mvn clean install` works with all these storages.  Right now I don't think that works for kafka and ispn storages.

Reply:
do you mean that doesn't work because of test failures? or what reason? Do you suggest to bring back all the profiles?

Reply:
We have Quarkus tests inside of `app` which are run not just when building app but also when building each of the storage modules.  Those Quarkus tests will fail for any storage type that doesn't support starting up an embedded instance of its required dependency (kafka, infinispan, etc).  The JPA storage works because we start up an H2 instance.  But I believe the ISPN and Kafka/Streams storage modules have not been updated to start embedded instances of their respective external deps yet.

Reply:
I'll try to verify this today.

Reply:
oh, ok, I was aware of that, yes kafka and streams doesn't manage storage dependencies, infinispan as ATM it uses embedded infinispan it just works. I could try to add kafka via test containers to app tests if you want